tests/jballerina-unit-test/src/test/resources/test-src/types/never/never-type.bal

```diff
@@ -0,0 +1,71 @@
+import ballerina/lang.'xml;
```

Comment:
Shall we add a test case for the following as well: 

>  A table shape satisfies a key-constraint key<K> if and and only if its set of key value shapes are a subset of K. The shape of a keyless table satisfies the key-constraint key<never>.

Reply:
Ack